src/Metrics.h

```diff
@@ -278,9 +295,12 @@ class TopN final : public Metric
     {
         auto section = json::array();
         auto items = _fi.get_frequent_items(datasketches::frequent_items_error_type::NO_FALSE_NEGATIVES);
+        auto threshold = _get_threshold();
```

Comment:
We call `get_frequent_items` here, and also in `_get_threshold`. I think it's a relatively expensive call so let's pass it into `_get_threshold` instead of calling it twice

Reply:
ok. done